data/languages.yml

```diff
@@ -0,0 +1,24 @@
+# Total lessons vs. completed lessons for each language
```

Comment:
These comments were just added initially for your reference. They are no longer accurate. Let's remove them.